scylla/src/statement/batch.rs

```diff
@@ -53,6 +53,7 @@ impl Batch {
     /// Sets the serial consistency to be used when executing this batch.
     /// (Ignored unless the batch is an LWT)
     pub fn set_serial_consistency(&mut self, sc: Option<Consistency>) {
```

Comment:
If we decide not to split the `Consistency` type, it would be nice if the `set_serial_consistency` functions would have a comment which warns that the function will panic if the consistency level is invalid.